tools/manually-handle-sharding-ddl-locks.md

```diff
@@ -0,0 +1,426 @@
+---
+title: Handle Sharding DDL Locks Manually
+summary: Learn how to handle sharding DDL locks manually.
+category: tools
+---
+
+# Handle Sharding DDL Locks Manually
+
+### Index
```

Comment:
@lilin90 we do not have `Index` section in English doc, right?
should we need to keep the format uniform?

Reply:
Yes, I'll fix it later